packages/blade-old/src/icons/index.js

```diff
@@ -51,6 +52,8 @@ export { default as PayoutLink } from './PayoutLink';
 export { default as PaymentPage } from './PaymentPage';
 export { default as Phone } from './Phone';
 export { default as Power } from './Power';
+export { default as ProductsFilled } from './ProductsFilled';
+export { default as ProductsOutlined } from './ProductsOutlined';
```

Comment:
```suggestion
export { default as GridFilled } from './GridFilled';
export { default as GridOutlined } from './GridOutlined';
```
Can we rename `ProductsFilled` to `GridFilled`? (also change at the component level)
Icon name is `Grid` in Figma & Grid seems generic rather than Products.

Reply:
+

Reply:
updated @gupta-ji6 @nikhilk1993